activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb

```diff
@@ -116,6 +116,10 @@ def supports_savepoints?
         true
       end
 
+      def supports_covering_indexes?
```

Comment:
Is this still needed?

Reply:
It seems no @simi. It never was. I just rebased and removed that method from that class.

Thanks!